### ISSUE TITLE
Cannot find module error

### ISSUE THREAD

**Issue by bpalermo** [wildlyinaccurate/angular-relative-date#20]
Hi,

I'm trying to use with angular 1.5.6 and browserify but I receive this errors:

```
Cannot find module './translations'
Cannot find module './filter'
```

Any ideas?

Thanks,
Bruno

**Comment by wildlyinaccurate** [comment]
How are you installing/using angular-relative-date?

**Comment by bpalermo** [comment]
I'm using with this boilerplate:

https://github.com/jakemmarsh/angularjs-gulp-browserify-boilerplate

Installed with:

```
npm i angular-relative-date --save-dev
```

```
import 'angular-relative-date'

const requires = [
  ...
  'relativeDate'
  ...
];

angular.module('app', requires);
```

**Comment by wildlyinaccurate** [comment]
Thanks, I'm able to reproduce this. I'll try and get to the bottom of it ASAP.

**Comment by wildlyinaccurate** [comment]
I've just published version 2.1.0 to npm. Could you please update and let me know if that resolves the issue?

**Comment by bpalermo** [comment]
Working... Thanks!
